From 16351810fe55292c54b2ef7c1b78469a0d89bae0 Mon Sep 17 00:00:00 2001
From: Guoqing Jiang <guoqing.jiang@suse.com>
Date: Thu, 8 Sep 2022 08:54:40 +0800
Subject: [PATCH] arm64: openeuler_defconfig: turn on Amazon ENA ethernet
 driver
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 16351810fe55292c54b2ef7c1b78469a0d89bae0
Modified-by-SEL: No


This driver is turned on on x86_64 already. Let's turn it on for arm64
as AWS supports ENA device on their arm64 instance.

Signed-off-by: Kai Liu <kai.liu@suse.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 593ac67497e3..92b33d2da470 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -2655,7 +2655,7 @@ CONFIG_NET_VENDOR_ALACRITECH=y
 # CONFIG_NET_VENDOR_ALTEON is not set
 # CONFIG_ALTERA_TSE is not set
 CONFIG_NET_VENDOR_AMAZON=y
-# CONFIG_ENA_ETHERNET is not set
+CONFIG_ENA_ETHERNET=m
 CONFIG_NET_VENDOR_AMD=y
 # CONFIG_AMD8111_ETH is not set
 # CONFIG_PCNET32 is not set
-- 
2.33.0

